wording tweaks

tkimnguyen opened the pull request.

adamcheasley commented on docs/about.rst:
```diff
@@ -102,12 +102,12 @@ Prefer keyword arguments
+#. There will never be any doubt when writing a method whether an argument should be positional or not.
    Decision already made.
-#. There will never be a doubt when using the API on which argument comes first, or which ones are named/positional.
+#. There will never be asy doubt when using the API on which argument comes first, or which ones are named/positional.
```
Typo here, "asy" should be "any"

tkimnguyen replied:
Done - thx for catching that!

gforcada commented:
@tkimnguyen would you mind rebasing so it can be merged?

tkimnguyen commented:
@gforcada er sorry, beyond my depth a bit here... rebase to what?

tkimnguyen commented:
I tried following the GitHub instructions here but it seemed to have no effect.

```
git checkout -b tkimnguyen-master master
git pull git://github.com/tkimnguyen/plone.api.git master

Step 2: Merge the changes and update on GitHub.

git checkout master
git merge --no-ff tkimnguyen-master
git push origin master
```

```
bash-3.2$ git status
On branch master
Your branch is up-to-date with 'origin/master'.
nothing to commit, working tree clean
bash-3.2$ git remote -v
origin	git@github.com:tkimnguyen/plone.api.git (fetch)
origin	git@github.com:tkimnguyen/plone.api.git (push)
bash-3.2$ git checkout -b tkimnguyen-master master

fatal: A branch named 'tkimnguyen-master' already exists.
bash-3.2$ bash-3.2$ git checkout tkimnguyen-master 
Switched to branch 'tkimnguyen-master'
bash-3.2$ git pull git://github.com/tkimnguyen/plone.api.git master
From git://github.com/tkimnguyen/plone.api
 * branch            master     -> FETCH_HEAD
Already up-to-date.
bash-3.2$ git checkout master
Switched to branch 'master'
Your branch is up-to-date with 'origin/master'.
bash-3.2$ git merge --no-ff tkimnguyen-master
Already up-to-date.
bash-3.2$ git push origin master
Everything up-to-date
bash-3.2$ 
```

gforcada commented:
@tkimnguyen forking rather than creating the branches in plone org itself makes it harder to help you on the rebase.

To shortcircuit it although I did the rebase myself and push a new branch in plone org: so I'm closing this and we can follow on https://github.com/plone/plone.api/pull/336

tkimnguyen commented:
thx @gforcada - I have been reluctant to mess with the main repo for fear of !@#$@ing something up but ok :)
